<launch>
  <node
      pkg="kuka_push"
      type="kukapush_env"
      name="kukapush_env"
      output="screen"
    />
</launch>
